apps/laboratory/src/utils/SiweUtils.ts

```diff
@@ -4,6 +4,8 @@ import { createSIWEConfig, formatMessage } from '@web3modal/siwe'
 import { WagmiConstantsUtil } from '../utils/WagmiConstants'
 
 export const siweConfig = createSIWEConfig({
+  signOutOnAccountChange: true,
```

Comment:
These are already `true` by default but added just in case to not miss these props in case we change them in the future.

We'd better to add e2e tests that disable these. So test runner should run the same tests without expecting SIWE sign when change network. I'm not sure how we could achieve this since these are the lab's settings.